doc/user/cluster_tls.md

```diff
@@ -35,8 +35,8 @@ The example cluster YAML manifest and example certs can be found in [example/tls
 
 The peer TLS assets should have the following:
 - **peer.crt**: peer communication cert.
-  The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc`.
-  In this case, it is `*.example.default.svc`.
+  The certificate should allow wildcard domain `*.${clusterName}.${namespace}.svc` and `*.{clusterName}.{namespace}.svc.cluster.local`.
```

Comment:
See https://github.com/coreos/etcd-operator/blob/master/pkg/util/etcdutil/member.go#L40 

After 0.5.1+, etcd operator will not have ".cluster.local" suffix in domain.

Reply:
tls was failing as the host didn't had .cluster.local suffix while creating the cert. I'll try with 0.5.2 and will let you know.